### ISSUE TITLE
ANSIBLE0009 false positive

### ISSUE THREAD

**Issue by Cinux90** [willthames/ansible-lint#331]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version:
ansible 2.4.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /bin/ansible
  python version = 2.7.5 (default, Aug  4 2017, 00:39:18) [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]

ansible-lint --version:
ansible-lint 3.4.17

```

- ansible installation method: yum
- ansible-lint installation method: yum (used epel-release)

# Desired Behaviour

Please look into below and help to fix the bug.
From my point of view, file(folder) permissions like 711 should be possible but ansible-lint show Errorcode: ANSIBLE0009

# Actual Behaviour (Bug report only)

On below little role I run ansible-lint and its failed with:
```
[root@localhost tests]# bash test-ansible.sh 
Examining test.yml of type playbook
Examining /root/testrole/tasks/main.yml of type tasks
[ANSIBLE0009] Octal file permissions must contain leading zero
/root/testrole/tasks/main.yml:2
Task/Handler: create test folder
```

I have a role in there I need to create a directory with permission 711. But its looks like every permission option <755 will result in error code ANSIBLE0009.

My minimal example looks like this:
task/main.yml:
```
---
- name: create test folder
  file:
    path: "/tmp/test"
    state: directory
    mode: 0711
```
tests/test-ansible.sh:
```
#!/bin/bash
ansible-lint -Rv "test.yml"
```
tests/test.yml:
```
---
- hosts: localhost
  remote_user: root
  roles:
    - testrole
```

**Comment by willthames** [comment]
This is fixed in source but not quite released. I'll sort that now

**Comment by willthames** [comment]
Version 3.4.21 is now released to pypi including this fix.

**Comment by Cinux90** [comment]
Thank you, Willthames. Its fixed now. Close the issue.
